ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_path_selection.py

```diff
@@ -80,6 +80,19 @@ def _get_path_groups(self) -> list:
             if pg_name in local_path_groups_names and path_group.get("ipsec", True):
                 path_group_data["ipsec_profile"] = self._cp_ipsec_profile_name
 
+            keepalive = path_group.get("dps_keepalive", {})
+            auto = keepalive.get("auto")
+            interval = keepalive.get("interval")
+            failure_threshold = get(keepalive, "failure_threshold", default=5)
+
+            if auto:
+                path_group_data["keepalive"] = {"auto": True}
+            elif interval is not None:
+                path_group_data["keepalive"] = {
+                    "interval": interval,
+                    "failure_threshold": failure_threshold,
+                }
```

Comment:
```suggestion
            if keepalive.get("auto") is not None:
                path_group_data["keepalive"] = {"auto": True}
            elif (interval := keepalive.get("interval")) is not None:
                path_group_data["keepalive"] = {
                    "interval": interval,
                    "failure_threshold": get(keepalive, "failure_threshold", default=5),
                }
```

Reply:
Changed input schema. Interval can now take auto or integer
Modified the code accordingly.

Reply:
I am not sure I understand this comment - I was just suggesting to refactor the code doing `get` on `auto` or `interval`. How did this trigger a schema change?

Reply:
lets keep it as it is I find it better like this :) but was surprised to see the changes!